GoogleForCreators/web-stories-wp
# UI Simplification: Saved Styles

## Summary
Removes the Saved Styles as a separate panel and moves it under Text Style panel, adding a popup.


## Relevant Technical Choices



## To-do

- [x] Add Confirmation dialogue for deleting a style.



## User-facing changes
| Before | After |
| ------ | ----- |
| <img width="312" alt="Screenshot 2021-11-08 at 19 15 59" src="https://user-images.githubusercontent.com/3294597/140787848-57d19663-7170-4f15-bf90-356e087e4ce3.png"> | <img width="576" alt="Screenshot 2021-11-08 at 19 13 40" src="https://user-images.githubusercontent.com/3294597/140787578-28391b41-5a56-4827-a68c-f067bb0ef078.png"> |
| <img width="308" alt="Screenshot 2021-11-08 at 19 16 05" src="https://user-images.githubusercontent.com/3294597/140787820-6eb2c19c-c9f9-4f9e-bf40-63d967abc224.png"> | <img width="270" alt="Screenshot 2021-11-08 at 19 13 47" src="https://user-images.githubusercontent.com/3294597/140787616-4551033a-078b-4518-a780-27654f459391.png"> |


## Testing Instructions






This PR can be tested by following these steps:

1. Add a text element
2. Verify the Saved Styles panel is not a separate panel anymore and is part of the Text Style panel
3. Verify that it's possible to add new styles by clicking on the + icon (add more than 2 styles).
4. Verify the panel only displays up to 2 styles in the design panel.
5. Verify that clicking on "More styles" opens a popup including all the added styles.
6. Verify that it's possible to apply styles by clicking on the style in the popup and in the panel
7. Verify that it's possible to delete styles in the popup by entering Edit mode (click on the Pencil icon) and then clicking on the style.


## Reviews

### Does this PR have a security-related impact?
No


### Does this PR change what data or activity we track or use?
No


### Does this PR have a legal-related impact?
No


## Checklist




---



Fixes #9463
